### PR TITLE
Mirego-base-setup 8 of X: CI tasks + lint

### PR DESCRIPTION
**Pull request by OlivierPineau**
## 📖 Description
This PR is updating and formatting the base setup code with Ktlint

Opening a PR to try and make the CI pass

### CODE REVIEW

File: androidApp/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -2,6 +2,7 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android">
 
     <uses-permission android:name="android.permission.INTERNET" />
+    <uses-permission android:name="android.permission.POST_NOTIFICATIONS" />
```
Comment (sdetilly) [inline androidApp/src/main/AndroidManifest.xml:5]: Why?

**Review by sdetilly — APPROVED** [review]

File: androidApp/src/main/AndroidManifest.xml (hunk 1 above)
Comment (marcantoinefortier) [reply androidApp/src/main/AndroidManifest.xml:5]: This should indeed not be part of the boilerplate IMO

Comment (OlivierPineau) [reply androidApp/src/main/AndroidManifest.xml:5]: Pretty sure it was crashing without it. I don't remember the reason though

Comment (sdetilly) [reply androidApp/src/main/AndroidManifest.xml:5]: Fixed, it was a problem with lint. Picasso can post notifications if you, for example, download an image. If you don't use that feature, you don't need to specify the permission, but you must tell lint to ignore it in the build.gradle file

**Pull request merged**